### PR TITLE
Explain permissions for the humans

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

File: assets/styles/stack.css [hunk 1]
```diff
@@ -89,7 +90,7 @@ body {
 
 /* Wrapper */
 .popup {
-    flex: 0 0 30rem
+    flex: 0 0 33rem
```
Comment (GoOz) [inline assets/styles/stack.css]: 30rem was too narrow ?

File: assets/styles/stack.css [hunk 2]
```diff
@@ -22,7 +22,8 @@ body {
     height: 100%;
     padding: 0;
     background-color: #fff;
-    font: 1em/1.5 Lato, sans-serif;
+    color: #333;
+    font: 1.05em/1.5 Lato, sans-serif;
```
Comment (GoOz) [inline assets/styles/stack.css]: why `1.05em` ?

File: assets/styles/stack.css (hunk 1 above)
Comment (nono) [inline assets/styles/stack.css]: I tried to be more respectful to the mockup made by Luc (and I thought it's looking better)

Comment (GoOz) [inline assets/styles/stack.css]: oh OK, I thought i measured it right the first, but nevermind.

**Pull request merged**